Bigtable/tests/Snippet/DataClientTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/*
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Tests\Snippet;
+
+use \Google\ApiCore\ServerStream;
+use Google\Cloud\Bigtable\DataClient;
+use Google\Cloud\Bigtable\V2\BigtableClient as TableClient;
+use Google\Cloud\Bigtable\V2\MutateRowsRequest_Entry;
+use Google\Cloud\Bigtable\V2\MutateRowsResponse;
+use Google\Cloud\Bigtable\V2\MutateRowsResponse_Entry;
+use Google\Cloud\Bigtable\V2\Mutation;
+use Google\Cloud\Bigtable\V2\Mutation_SetCell;
+use Google\Cloud\Core\Testing\Snippet\SnippetTestCase;
+use Google\Cloud\Core\Testing\TestHelpers;
+use Google\Rpc\Code;
+use Google\Rpc\Status;
+use Prophecy\Argument;
+
+/**
+ * @group bigtable
+ * @group bigtabledata
+ */
+class DataClientTest extends SnippetTestCase
+{
+
+    const PROJECT_ID = 'my-project';
+    const INSTANCE_ID = 'my-instance';
+    const TABLE_ID = 'my-table';
+    const TABLE_NAME = 'projects/my-project/instances/my-instance/tables/my-table';
+
+    private $bigtableClient;
```

Comment:
Just a nit: I think it's fine to group all of the below variables together.
Looks like a blank line between each of them is not quite necessary.

Reply:
done